run.sh

```diff
@@ -1,5 +1,27 @@
 #!/bin/bash
 
+function publishToTurquoise() {
+    status=$1
+    sample=$2
+    event='{ \
+      "timestamp": "%s", \
+      "type": "snp_check.complete", \
+      "subjects": [ \
+        { \
+          "name": '"$sample"', \
+          "type": "sample" \
+        } \
+      ], \
+      "labels": [ \
+        {
+          "name": "status", \
+          "value": "'$status'" \
+        } \
+      ] \
+    }'
```

Comment:
I don't think this will work, the entire payload is enclosed in single quotes so I don't think you'll get variable substitution in the entire block. You'll have to instead put double-quotes around everything and escape all the "inside" double quotes.